executor/runtime/types/sidecars.go

```diff
@@ -124,6 +125,12 @@ var sideCars = []ServiceOpts{
 			"/titus/container-tools": {},
 		},
 	},
+	{
+		ServiceName:  SidecarTrafficSteering,
+		UnitName:     "titus-sidecar-traffic-steering",
```

Comment:
I don't see this unit. 

Is this really a new systemd titus system service?

Reply:
This has been added